pkg/workloads/cortex/serve/serve.py

```diff
@@ -38,6 +38,7 @@
 from cortex.lib.concurrency import LockedFile
 from cortex.lib.storage import S3, LocalStorage
 from cortex.lib.exceptions import UserRuntimeException
+from cortex.lib.concurrency import FileLock
```

Comment:
We can just merge this line with `from cortex.lib.concurrency import LockedFile` so that it ends up being `from cortex.lib.concurrency import LockedFile, FileLock`.